Makefile

```diff
@@ -17,9 +17,14 @@ endif
 # Targets
 #--------------------------------------------------
 install: 
+	@echo "Installing local runner"
 	@(cd ./local-runner && npm install)
+	@echo "Installing R env packages"
+	@(cd ./pipeline-runner && R -e "renv::restore()")
```

Comment:
```suggestion
	@(cd ./pipeline-runner && R -e "renv::restore()")
	@(cd ./pipeline-runner && R -e "renv::install()")
```

Reply:
Also install interactive development dependencies, so that make test can be run.